include/lbann/layers/io/target/generic_target_layer.hpp

```diff
@@ -53,6 +53,9 @@ class generic_target_layer : public io_layer {
       m_paired_input_layer(input_layer),
       io_buffer(nullptr),
       m_ground_truth(nullptr) {
+    // Target layers have two parents, the layer that it will feed
+    // back the response to, and the input layer where it gets the response
+    m_expected_num_parent_layers = 2;
     // Target layers have no children
     m_expected_num_child_layers = 0;
```

Comment:
This is fine for now, but we should expect a child layer if we want the target layer to output to other models.